File: lib/pg-client.js
const { Client } = require('pg');

const client = new Client({
  user:     'postgres',
  host:     'localhost',
  database: 'postgres',
  port:     54320,
});

client.connect();

async function executeSql(statement) {
  console.debug(statement);
  return client.query(statement)
    .catch((err) => {
      console.error('ERROR', err);
    });
}

module.exports = {
  executeSql,
};

File: analyzer/pg_client.py
import psycopg2

connection = psycopg2.connect(user="postgres",
                              host="127.0.0.1",
                              port="54320",
                              database="postgres")

def execute_sql(query):
  cursor = connection.cursor()
  cursor.execute(query)
  results = cursor.fetchall()
  cursor.close()
  return results
